strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/storage/repository/CustomConfiguration.java

```diff
@@ -2,13 +2,15 @@
 
 import org.carlspring.strongbox.xml.CustomTag;
 
+import javax.persistence.Entity;
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlRootElement;
 
 /**
  * @author carlspring
  */
+@Entity
```

Comment:
@fuss86 I think in this case it's better to be `@Embeddable` as you suggested instead of `@Entity` :)

Reply:
Yeah, both `@Entity` and `@Embeddable` could be then picked up by `oEntityManager.registerEntity` method. Thanks  :+1: Will change it in the next PR.

Reply:
yeah, great :)